code/datums/spawners_menu.dm

```diff
@@ -52,7 +52,12 @@
 		for(var/mob/joinable_mob as anything in GLOB.joinable_mobs[mob_type])
 			this["amount_left"] += 1
 			if(!this["desc"])
-				this["desc"] = initial(joinable_mob.desc)
+				if(HAS_TRAIT(joinable_mob, TRAIT_GHOSTROLE_ON_REVIVE))
+					this["you_are_text"] = "You are a long dead crewmember, but are soon to be revived to rejoin the crew!"
+					this["flavor_text"] = "Get a job and get back to work!"
+					this["important_text"] = "Do your best to help the station. You still roll for midround antagonists."
```

Comment:
This feels weird to have hardcoded right here in the spawners menu
I feel it'd be better to have some more generic method of letting the mob set this data like a signal, and have the component handle it
Avoids this becoming a weird if-else chain of unrelated checks for specific ghost roles